packages/terra-functional-testing/src/services/wdio-selenium-docker-service.js

```diff
@@ -92,14 +93,14 @@ class SeleniumDockerService {
 
     await exec(`${envVars}docker-compose -f ${this.getDockerComposeFilePath()} up -d`);
     await this.waitForSeleniumHubReady();
+
+    logger.info('Successfully started the docker selenium hub.');
```

Comment:
I added this log so we can see when the docker hub is active. Sometimes in large projects webpack takes a while to complete and because the last log was "starting docker..." it seems like it's waiting on docker, but it's actually waiting on webpack. This just adds verification the docker hub started.